
syntax = "proto3";

package mesop.components.radio;

message RadioType {
  string color = 1;
  string name = 2;
  string label_position = 3;
  string value = 4;
  bool disabled = 5;
  bool required = 6;
  string on_radio_change_event_handler_id = 20;
  repeated RadioOption radio_options = 7;
}

message RadioOption {
  string label = 1;
  string value = 2;
}
